sanic/response.py

```diff
@@ -73,28 +73,121 @@
 }
 
 
-class HTTPResponse:
+class BaseHTTPResponse:
+    def _encode_body(self, data):
+        try:
+            # Try to encode it regularly
+            return data.encode()
+        except AttributeError:
```

Comment:
No this is for the case where it's a str in the first place, else it could be something like an int hence the AttributeError